app/konnect/org-management/teams-and-roles/manage.md

```diff
@@ -43,6 +43,8 @@ Edit team roles from the {% konnect_icon organizations %} [**Organization**](htt
 
 1. Click **Add role(s)**.
 
+1. Select the region you want to add from the **Region** drop-down menu. Members assigned to this role can only access {{site.konnect_short_name}} objects in the selected region. 
```

Comment:
```suggestion
1. Select the region you want to add from the **Region** drop-down menu. Members assigned to this role can only access {{site.konnect_short_name}} objects in the selected region. 
```

Would this be better as a note after the last step?

Reply:
I put it there because I thought users might want this info as they are configuring that step. Do you typically put this type of "additional" info after the last step though?